components/keyboard-shortcuts/README.md

```diff
@@ -36,8 +36,6 @@ class SelectAllDetection extends Component {
 }
 ```
 
-__Note:__ The value of each shortcut should be a consistent function reference, not an anonymous function. Otherwise, the callback will not be correctly unbound when the component unmounts.
```

Comment:
I haven't found this to be the case. As presented here, the callback is irrelevant to the `unbind` behavior.